notebook/static/notebook/js/actions.js

```diff
@@ -444,7 +451,7 @@ define(function(require){
                 return true;
             }
         },
-        'move-cursor-up-or-previous-cell':{
```

Comment:
I'm not too strongly attached to this one. I'm even wondering if we shouldn't hide some action from CP.